microk8s-resources/actions/enable.reg.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -e
+
+source $SNAP/actions/common/utils.sh
+
+read -ra ARGUMENTS <<< "$1"
+argz=("${ARGUMENTS[@]/#/--}")
```

Comment:
I do not think you are doing anything with `argz` so lets simplify the code by removing these two lines.

Reply:
Alright!